argusclient/client.py

```diff
@@ -741,7 +746,7 @@ def check_success(resp, decCls):
             raise ArgusException(resp.text)
         return res
     elif resp.status_code == httplib.NOT_FOUND:
-        raise ArgusException("Object not found at endpoint: %s message: %s" % (resp.request.url, resp.text))
+        raise ArgusObjectNotFoundException("Object not found at endpoint: %s message: %s" % (resp.url, resp.text))
     elif resp.status_code == httplib.UNAUTHORIZED:
         raise ArgusAuthException("Failed to authenticate at endpoint: %s message: %s" % (resp.request.url, resp.text))
```

Comment:
Could you change here also from `resp.reqeust.url` to `resp.url` for consistency sake?